### ISSUE TITLE
About Scribus section on home page is too wordy and technical

### ISSUE THREAD

**Issue by GarryPatchett** [nitramr/scribus-webpage#11]
Needs to be re-written as something more "enthusiastic", "upbeat", and "active", and far less technical.
I have no idea what to put there just yet though.

**Comment by nitramr** [comment]
I have copied text paragraphs from: http://write.flossmanuals.net/scribus/introduction/

https://github.com/nitramr/scribus-webpage/commit/ceeaf93b0718ff017ebd2e47a74df521ec35cec6

**Comment by GarryPatchett** [comment]
Much better than what was there before.
